packages/xstate-fsm/test/fsm.test.ts

```diff
@@ -131,6 +131,88 @@ describe('@xstate/fsm', () => {
     expect(nextState.actions).toEqual([]);
   });
 
+  describe('when a wildcard transition is defined', () => {
+    type ParserContext = { entries: Array<Record<string, string>> };
```

Comment:
I'd copy-paste this stuff into each added test and trim down redundant bits for each particular test. Don't be afraid of some repetition, embrace it! 😉